tests/test_integration/stencil_definitions.py

```diff
@@ -220,6 +220,17 @@ def horizontal_diffusion(in_field: Field3D, out_field: Field3D, coeff: Field3D):
         )
 
 
+@register
+def large_k_interval(in_field: Field3D, out_field: Field3D):
+    with computation(PARALLEL):
+        with interval(0, 6):
+            out_field = in_field
+        with interval(6, -10):  # this stage will only run if field has more than 16 elements
```

Comment:
Does this mean that interval will be skipped or it will crash? I think this comment could be expanded in the proper test definition

Reply:
It will be skipped and this part of the test. I will add a comment in the test.